EDI/Operation/01_ManageDefinitions.md

```diff
@@ -0,0 +1,249 @@
+
+- At least one connection has been created, see [Create connection](../Integration/01_ManageConnections.md#create-connection).
+- At least one definition setting has been created, see [Create basic definition settings](#create-basic-definition-settings).
+- You have defined the file settings for this definition, see [Manage header file settings](./02_ManageHeaderFileSett.md#manage-header-file-settings) and [Manage line item file settings](./04_ManageLineItemsFileStt.md#manage-line-item-file-settings).
+- You have defined the data sources for this definition, see [Assign data source](./05_ManageDataSources.md#assign-data-source).
+- You have defined the mapping settings for this definition, see [Manage mapping settings](./06_ManageMappingSettings.md).
+- You have prepared the related workflow to send the messages when they are needed in the process, see [Prepare workflow](../Operation/07_PrepareWorkflow.md).
+
+#### Procedure
+
+*DataHub Exporter > Exports*
+
+![Exporter definitions](../../Assets/Screenshots/EDI/Operation/ExportDefinitions.png "[Exporter definitions]")
+
+1. Click the definition you want to activate.   
+    The *Edit definition "definition name"* view is displayed. The *Settings* tab is displayed by default.
+
+    ![Edit definition](../../Assets/Screenshots/EDI/Operation/ExportDefinitionEdit.png "[Edit definition]")
+
+2. Click the *Active* toggle.   
+    The toggle is highlighted.
+
+    ![Activate definition](../../Assets/Screenshots/EDI/Operation/ExportDefinitionActivate.png "[Activate definition]")
+
+3. Click the [SAVE] button.  
+    The *Exporter definitions* view is displayed. The status in the *Active* column has switched to *Active*. The definition is activated, and you can send messages now. 
+
+4. Continue with preparing the workflows. For detailed information, see [Prepare workflow](./07_PrepareWorkflow.md).
+
+
+## Deactivate definition
+
+Deactivate a definition if it is no longer needed. We recommend deactivating a definition that is no longer needed instead of deleting it.
+
+#### Prerequisites
+
+- At least one definition setting has been activated, see [Activate definition](#activate-definition).   
+- You no longer want to send messages of this type to your business partner.
+
+#### Procedure
+
+*DataHub Exporter > Exports*
+
+![Exporter definitions](../../Assets/Screenshots/EDI/Operation/ExportDefinitions.png "[Exporter definitions]")
+
+1. Click the definition you want to deactivate.   
+    The *Edit definition "definition name"* view is displayed. The *Settings* tab is displayed by default.
+
+    ![Edit definition](../../Assets/Screenshots/EDI/Operation/ExportDefinitionActivate.png "[Edit definition]")
+
+2. Click the *Active* toggle.   
+    The toggle is no longer highlighted.
+
+3. Click the [SAVE] button.  
+    The *Exporter definitions* view is displayed. The definition is listed as **Inactive** in the *Active* column. You can no longer send messages of this type to your business partner. 
```

Comment:
Bei Activate definition, eine andere Formulierung. Konsistent machen?
The status in the *Active* column has switched to **Active**.